<launch>

<arg name="nums"  default="3"/>
<arg name="max_planning_time"  default="2.0"/>
<arg name="task_step"  default="0.1"/>
<arg name="index"  default="1"/>


<node pkg="dual_rrt_star" type="compare_node" name="compare_node" output="screen" required = "true">
	<param name="dualrrt/nums"       		   value="$(arg nums)" />
	<param name="dualrrt/max_planning_time"      value="$(arg max_planning_time)" />
	<param name="dualrrt/task_step"              value="$(arg task_step)"/>
	<param name="dualrrt/constrain_index"        value="$(arg index)"/>
</node>

</launch>

